python/tests/test_aac/context/test_definition_parser.py

```diff
@@ -0,0 +1,44 @@
+from unittest import TestCase
+
+from aac.context.language_context import LanguageContext
+from aac.context.definition_parser import DefinitionParser
+from aac.in_out.parser._parse_source import parse
+
+
+class TestDefinitionParser(TestCase):
+    def test_load_definitions(self):
+        parser = DefinitionParser()
+        context = LanguageContext()
+        definitions = parse(VALID_AAC_YAML_CONTENT)
+        loaded_definitions = parser.load_definitions(context=context, parsed_definitions=definitions)
+        self.assertIsNotNone(definitions[0].instance)
+        self.assertEqual(len(loaded_definitions), 1)
+        self.assertEqual(definitions[0].name, "TestSchema")
+        self.assertEqual(loaded_definitions, definitions)
+        self.assertEqual(len(definitions[0].instance.fields), 4)
+
+
+
+VALID_AAC_YAML_CONTENT = """
```

Comment:
Remember we were wanting to have both valid and invalid testing moving forward to help make the tests more robust. I would go ahead and add an invalid yaml and basic testing. 

That, looking this test over, the other language context tests, and thinking through a test approach more, I don't think we would have testing against the internal/individual methods like I had previously commented in a discussion of this task. You are able to test against the results those internal pieces generate because they are rolled up to the top level `load_definitions` method. So, just evaluating from there should be fine as long as we are checking the returned contents thoroughly.

Reply:
Think this test failure works for now, as we break it up might tweak it a bit.